kura/org.eclipse.kura.core.comm/OSGI-INF/comm.xml

```diff
@@ -10,9 +10,10 @@
 
     Contributors:
       Eurotech
+      Red Hat Inc - clean up
 
 -->
-<scr:component xmlns:scr="http://www.osgi.org/xmlns/scr/v1.1.0" activate="activate" deactivate="deactivate" immediate="true" name="org.eclipse.kura.core.comm.CommConnectionFactory">
+<scr:component xmlns:scr="http://www.osgi.org/xmlns/scr/v1.1.0" immediate="true" name="org.eclipse.kura.core.comm.CommConnectionFactory">
```

Comment:
It's better to have this service component delayed as no other service reference is actually consumed by this component. It can be activated on demand.